include/swift/AST/DiagnosticsSema.def

```diff
@@ -4136,6 +4136,15 @@ ERROR(fixed_layout_attr_on_internal_type,
       "%select{private|fileprivate|internal|%error|%error}1",
       (DeclName, AccessLevel))
 
+WARNING(fixed_layout_struct,
+      none, "'@frozen' attribute is now used for fixed-layout structs", ())
```

Comment:
Nitpick: please pick either `'@frozen'` or `'frozen' attribute` (both here and below); having both is redundant.